src/ducks/lagret-filter-ui-state.ts

```diff
@@ -0,0 +1,111 @@
+import {OrNothing} from "../utils/types/types";
+import {LagretFilter} from "./lagretFilter";
+
+// Actions
+export const VELG_LAGRET_FILTER = 'lagretfilter_velg/VELG_LAGRET_FILTER';
+export const MARKER_LAGRET_FILTER = 'lagretfilter_velg/MARKER_LAGRET_FILTER';
+export const AVMARKER_LAGRET_FILTER = 'lagretfilter_velg/AVMARKER_LAGRET_FILTER';
+export const AVMARKER_SISTE_VALGT_LAGRET_FILTER = 'lagretfilter_velg/AVMARKER_SISTE_VALGT_FILTER';
+
+export const MARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/MARKER_VEILEDER_GRUPPE';
+export const AVMARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/AVMARKER_VEILEDER_GRUPPE';
+
+export const APEN_LAGRE_FILTER_MODAL = 'lagretfilter_velg/APEN_LAGRE_FILTER_MODAL';
+export const LUKK_LAGRE_FILTER_MODAL = 'lagretfilter_velg/LUKK_LAGRE_FILTER_MODAL';
+
+
+export interface LagretFilterUIState {
+    valgtLagretFilter: OrNothing<LagretFilter>;
+    valgtVeilederGruppe: OrNothing<LagretFilter>;
+    sisteValgtLagretFilter: OrNothing<number>
+    erModalApen: boolean;
+}
+
+const initialState = {
+    valgtLagretFilter: null,
+    valgtVeilederGruppe: null,
+    sisteValgtLagretFilter: null,
+    erModalApen: false,
+};
+
+//  Reducer
+export default function reducer(state: LagretFilterUIState = initialState, action): LagretFilterUIState {
+    switch (action.type) {
+        case
+        MARKER_LAGRET_FILTER:
+            return {...state, valgtLagretFilter: action.data, sisteValgtLagretFilter: action.data.filterId}
+        case
+        AVMARKER_LAGRET_FILTER:
```

Comment:
Change this to AVMARKER_MINE_FILTER since it's not both mine filter and veiledergrupper, only mine filter